### ISSUE TITLE
Add import and export

### ISSUE THREAD

**Issue by keneanung** [AchaeaBashingScript/Bashing#20]
Importing and exporting priorities might be useful. But we need to look into handling conflicts...

**Comment by zylvyn** [comment]
What kinds of conflicts were you anticipating?  Pre-existing entries?

**Comment by keneanung** [comment]
Exactly. And come think of it, conflicts were not considered in #40 either.

**Comment by zylvyn** [comment]
Actually, they were.

```
if not table.contains(keneanung.bashing.configuration.priorities[area],denizenString) then
	keneanung.bashing.configuration.priorities[area][#keneanung.bashing.configuration.priorities[area]+1] = denizenString
end
```

That only imports them if they're not already in the prio table already.  Leaves them where they are if they exist, adds them to the end if they do not.

**Comment by keneanung** [comment]
*goes back into the dark basement room* right...

**Comment by zylvyn** [comment]
So were you thinking to let them export to a file (that they name) and import from a file (that they name)?  Or simply have a single exported-to file that can be shared/backed up and imported from (if put into the relevant folder)?  Maybe include a starter package in the .mpackage?

**Comment by keneanung** [comment]
I was thinking about the named file variant. I didn't think about a starter file yet, to be honest.

**Comment by zylvyn** [comment]
[This](https://github.com/AchaeaBashingScript/Bashing/pull/41) sort of what you had in mind?

**Comment by keneanung** [comment]
Closed by #41
